src/Discord.Net.Examples/Discord.Net.Examples.csproj

```diff
@@ -15,7 +15,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Discord.Net.Core\Discord.Net.Core.csproj" />
     <ProjectReference Include="..\Discord.Net.WebSocket\Discord.Net.WebSocket.csproj" />
-    <PackageReference Include="JetBrains.Annotations" Version="2018.3.0" />
+    <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
```

Comment:
Why do we have these, anyways?

Reply:
The PublicAPI attribute is being used for a number of the examples, so I'm guessing its for the docs.